package/test/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"compilerOptions": {
+		"target": "ESNext",
+		"watch": true,
+		"lib": [
+			"ESNext",
+			"DOM"
+		],
+		"module": "ESNext",
+		"moduleResolution": "node",
+		"sourceMap": true,
+		"skipLibCheck": true,
+		"allowSyntheticDefaultImports": true
+	},
+	"exclude": [
+		"node_modules",
+		"./node_modules",
+		"./node_modules/*"
```

Comment:
```suggestion
		"node_modules"
```

Reply:
Sorry, didn't see that, I just copied the tsconfig from another project of mine and forgot to remove those